client/components/tinymce/plugins/simple-payments/dialog/product-image.jsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import MediaUtils from 'lib/media/utils';
+import QueryMedia from 'components/data/query-media';
+import { getMediaItem } from 'state/selectors';
+
+const ProductImage = ( { siteId, imageId, image } ) => {
+	if ( ! siteId || ! imageId ) {
+		return null;
+	}
+
+	if ( ! image ) {
+		return (
+			<figure className="editor-simple-payments-modal__list-figure is-placeholder">
```

Comment:
Good catch, thanks! It should be `...__figure` instead of `...__list-figure`.

There is a placeholder (a pulsing gray rectangle) displayed while the image is loading. At the beginning, we know only the ID of the image in the media library. To retrieve its URL, we need to issue a REST API request. Only then we can render an `<img src>` element.